__sdk__.js

```diff
@@ -69,6 +69,10 @@ module.exports = {
   fastlane: {
     entry: "./src/connect/interface",
   },
+  "three-domain-secure": {
+    entry: "./src/three-domain-secure/interface",
+    globals,
```

Comment:
we probably don't need these

Reply:
@wsbrunson - I just wanted to get clarification on this comment you made on this. 

Were you saying that `globals` isn't needed or the entire `three-domain-secure` object in this file? 

The entire `three-domain-secure` object needs to be passed for it to be available in the `components=three-domain-secure` query param, right?